app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,22 @@
+.show-container {
+  padding-top: 50px;
+  margin: 10px;
+  padding: 5px;
+}
+
+.show-title {
+ margin-bottom: 0px;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>`0px` should be written without units as `0`